src/metrics.go

```diff
@@ -79,10 +79,12 @@ func qCacheHitRatio(metrics map[string]interface{}) (float64, bool) {
 }
 
 func slaveRunningAsNumber(metrics map[string]interface{}) (int, bool) {
-	slaveRunning, ok := metrics["Slave_running"].(string)
-	if ok && slaveRunning == "ON" {
-		return 1, true
-	} else if ok && slaveRunning == "OFF" {
+	slaveIORunning, okIO := metrics["Slave_IO_Running"].(string)
+	slaveSQLRunning, okSQL := metrics["Slave_IO_Running"].(string)
+	if okIO || okSQL {
+		if slaveIORunning == "Yes" || slaveSQLRunning == "Yes" {
```

Comment:
Do we `or` them here because we are fine having either? If so, I would put a comment just to clarify that both are equally fine and none is preferred to the other.

Reply:
This is supposed to be AND